charts/gitness/templates/hpa.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
```

Comment:
HPA isn't supported due to the code's lack of a distributed architecture. Can we remove this file?

Reply:
Is removed :)